build-release.ini

```diff
@@ -0,0 +1,17 @@
+; SPDX-FileCopyrightText: 2024 Emil Velikov <emil.l.velikov@gmail.com>
+; SPDX-FileCopyrightText: 2024 Lucas De Marchi <lucas.de.marchi@gmail.com>
+;
+; SPDX-License-Identifier: LGPL-2.1-or-later
+
+[project options]
+build-tests = true
```

Comment:
Asserts are disabled in release builds of meson v0.59+ IIRC, where bunch (but not all) of our tests hard-depend on them.

So let's make that `false` + it will spare hunting for extra dependencies which are currently missing from the README.